<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPrisoner</class>
 <widget class="QDialog" name="EditPrisoner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;

}

#EditPrisoner{
	background-color:#001F3F;
}


QLineEdit {
    min-height: 30px;
    min-width: 250px;
    font-size: 16px;
}

#label,#label_2,#label_3,#label_4,#label_5{
	font-size:16px;
	color:white;
}


#edit_btn{
	background-color : #cb0000 ;
	border-radius : 10px;
	color:white;
	font-size:15px;
	padding:10px;
}

#edit_btn:hover{
	background-color: #850101;
}

#label_6{  /*edit label */
    font-size: 20px;      /* Adjust the font size */
    font-family: 'Segoe UI';  /* Change the font */
    color-white;
	font-weight:bold;
 
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Edit Prisoner:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ageLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Crime:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="crimeLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Crime Details:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="crimeDetailsTextEdit">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Sentence Time:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="sentenceLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="edit_btn">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
